content/recipes/terminus.md

```diff
@@ -1,114 +1,137 @@
-### Health checks (Terminus)
+### Healthchecks (Terminus)
 
-The [terminus](https://github.com/godaddy/terminus) offers hooks to react on graceful shutdowns and supports you creating proper [Kubernetes](https://kubernetes.io/) readiness / liveness checks for any HTTP application. The module [@nestjs/terminus](https://github.com/nestjs/terminus) integrates the terminus library with the Nest ecosystem.
+The NestJS **Terminus** integration supports you with **readiness / liveness** health checks. Healthchecks are very important when it comes to complex
+backend setups. In a nutshell, a health check in the realm of web development usually consists of a special address, for example, `https://my-website.com/health/readiness`.
+A service or similar (e.g. Kubernetes) checks this address continuously. Depending on the HTTP status code of this address the service will take action once it is an "unhealthy" response.
```

Comment:
Suggestion:
once it is an "unhealthy" response.
->
when it receives an "unhealthy" response.